src/main/java/com/leafbound/services/CartService.java

```diff
@@ -0,0 +1,18 @@
+package com.leafbound.services;
+
+import com.leafbound.models.Carts;
+
+public interface CartService {
+	
+	//find products by id
+	public Products findProdById(int id);
+	
+	// add
+	public boolean addtoCart(Carts cart);
+	
+	//update cart
+	public boolean updateCart(Carts cart);
```

Comment:
Because all we're doing with cart is either `add` or `delete` items in the cart, we don't need an `update`.

You can comment it out for now.